test-cases/longevity/longevity-100GB-48h-cloud-CloudLimitedChaosMonkey-tls.yaml

```
@@ -3,7 +3,7 @@ prepare_write_cmd: "cassandra-stress write cl=ALL n=100100150  -schema 'replicat
 
 stress_cmd: ["cassandra-stress write cl=QUORUM duration=2860m  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=250  -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=400200300..500200300 -log interval=15"]
 stress_read_cmd: ["cassandra-stress read cl=ONE duration=2860m -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=250  -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..100100150 -log interval=5"]
-run_fullscan: '{"ks_cf": "keyspace1.standard1", "interval": 5}' # 'ks.cf|random, interval(min)'
+run_fullscan: '{"mode": "select", "ks_cf": "keyspace1.standard1", "interval": 5}' # 'ks.cf|random, interval(min)'
```

you changed this parameter to be a list, so the way it is here, it won't work, as it is a string...
you need either to use in `sct_config.py` the type to be `str_or_list`, or change this one to be a list of 1 string:

```suggestion
run_fullscan: 
    - '{"mode": "select", "ks_cf": "keyspace1.standard1", "interval": 5}' # 'ks.cf|random, interval(min)'
```

---

I changed it (and all other single ones) to be lists, for consistency.